src/test/java/org/broadinstitute/consent/http/service/InstitutionServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package org.broadinstitute.consent.http.service;
+
+import org.broadinstitute.consent.http.db.InstitutionDAO;
+import org.broadinstitute.consent.http.models.Institution;
+import org.junit.Assert;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import org.junit.Before;
+import org.junit.Test;
+import static org.mockito.Matchers.anyInt;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import org.mockito.Mock;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.doThrow;
+
+import org.mockito.MockitoAnnotations;
+
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class InstitutionServiceTest {
+
+  private InstitutionService service;
+
+  @Mock
+  private InstitutionDAO institutionDAO;
+
+  @Before
+  public void setUp() {
+    MockitoAnnotations.initMocks(this);
+  }
+
+  private void initService() {
+    service = new InstitutionService(institutionDAO);
+  }
+
+  @Test
+  public void testCreateInstitution() {
+    initService();
+    //mock findByInstitution (not insertInstitution) because that is the call that service.createInstitution returns
+    when(institutionDAO.findInstitutionById(anyInt())).thenReturn(getInstitutions().get(0));
+    Institution institute = service.createInstitution(anyString(), anyString(), anyString(), anyInt(), eq(new Date()));
+    assertEquals(getInstitutions().get(0), institute);
+  }
+
+  @Test
+  public void testUpdateInstitutionById() {
+    initService();
+    //doNothing is default for void methods, no need to mock InstitutionDAO.updateInstitutionById
+    try {
+      service.updateInstitutionById(anyInt(), eq("New Name"), anyString(), anyString(), anyInt(), eq(new Date()));
+    } catch (Exception e) {
+      Assert.fail("Update should not fail");
+    }
+  }
+
+  @Test
+  public void testUpdateInstitutionByIdFail() {
+    initService();
+    doThrow(new RuntimeException("Update method should pass on error from DAO")).when(institutionDAO).updateInstitutionById(anyInt(), anyString(), anyString(), anyString(), anyInt(), eq(new Date()));
+  
+    try {
+      service.updateInstitutionById(anyInt(), eq("New Name"), anyString(), anyString(), anyInt(), eq(new Date()));
+    } catch (Exception e) {
+      assertEquals(e.getMessage(), "Update method should pass on error from DAO");
+    }
+  }
```

Comment:
Instead of wrapping the call in a try, you can add an `expected` to the test annotation. In testing scenarios, prefer checked vs runtime exceptions
```suggestion
  @Test(expected = Throwable.class)
  public void testUpdateInstitutionByIdFail() {
    initService();
    doThrow(new Exception("Update method should pass on error from DAO")).when(institutionDAO).updateInstitutionById(anyInt(), anyString(), anyString(), anyString(), anyInt(), eq(new Date()));
    service.updateInstitutionById(anyInt(), eq("New Name"), anyString(), anyString(), anyInt(), eq(new Date()));
  }
```